### DISCUSSION TITLE
Hide unused inputs on the client

### DISCUSSION THREAD

**Discussion by mahdiyari** [scalar/scalar#3755] [Ideas]
Would be nice to have options to hide unused inputs. The inputs highlighted in the image are never used for any of my APIs. So hiding them would make the UI better.

![image](https://github.com/user-attachments/assets/ed6bd758-c7d1-4656-9121-3a26e1c45562)

**Comment by antlio** [comment]
hey @mahdiyari, thanks for that great feedback! do you feel like the modal should be as close as possible from the reference? that's actually a topic we talked about recently!

will follow up here regarding what we do about this as soon as we dig that up.

**Comment by mahdiyari** [comment]
More customization the better. "Test Request" is a nice addition to the documentation but it seems too generic.
